java/kotlin-extractor/src/main/kotlin/KotlinUsesExtractor.kt

```diff
@@ -1191,20 +1191,22 @@ open class KotlinUsesExtractor(
         else
             f.parentClassOrNull?.let { parentClass ->
                 getJavaEquivalentClass(parentClass)?.let { javaClass ->
-                    if (javaClass != parentClass)
+                    if (javaClass != parentClass) {
+                        val jvmName = getSpecialJvmName(f) ?: f.name.asString()
```

Comment:
Are you sure we don't want `getJvmName`?

Reply:
It doesn't make a difference, because these functions have no annotation on them. I pushed a change to use `getJvmName`, and reverted its visibility change.